src/components/PageHelp.tsx

```diff
@@ -237,23 +237,37 @@ const Help = () => (
             Dashboard is built using React.js, Patternfly and other JS goodies.
         </p>
         <p>
-            The project is currently developed on {"CEE's"} gitlab.
+            The project is currently developed on github.
             Contributions are always welcome.
         </p>
-        <p className="padding-left-20">
-            <a href="https://gitlab.cee.redhat.com/osci/ci-dashboard">
-                https://gitlab.cee.redhat.com/osci/ci-dashboard
-            </a>
-        </p>
-        <p>
-            {"Dashboard's"} backend - Kai - is developed separately. You can
-            find the code here:
-        </p>
-        <p className="padding-left-20">
-            <a href="https://gitlab.cee.redhat.com/baseos-qe/kai">
-                https://gitlab.cee.redhat.com/baseos-qe/kai
-            </a>
-        </p>
+        <table style={{ width: '45%' }}>
```

Comment:
I'm not a big fan of tables for this use case. I'd prefer to use a [list](https://www.patternfly.org/v4/components/list/) here. You can also use the [`ExternalLink` component](https://github.com/fedora-ci/ciboard/blob/2770c41f5cb3718f50a60bb57e150f9b2d8e93b6/src/components/ExternalLink.tsx) for the links.

Reply:
I would've used list, but I found this kind of alignment nicer visually.